modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionTypeGlobalJSShouldFail/with-invalid-restrictedToAdminPages/client-extension.yaml

```diff
@@ -0,0 +1,4 @@
+with-invalid-restrictedToAdminPages:
```

Comment:
I see `with-invalid-script-location` (all kebab-case) in another file, but you're using camelCase here. I'd say make it all kebab-case for consistency.